core/dplug/core/ringbuf.d

```diff
@@ -135,14 +135,13 @@ struct RingBufferNoGC(T)
         }
 
         ///
-        int opApply(scope int delegate(ref T) dg)
```

Comment:
Fixed by means of: No *access by reference* anymore. Should not be needed for most (if not *all*) use-cases anyway.